.github/workflows/release.yml

```diff
@@ -9,24 +9,20 @@ jobs:
     strategy:
```

Comment:
@cb22 I just noticed that this workflow is never run because the release is triggered by the GITHUB_TOKEN job.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow

TIL when this token creates things, events are not triggered. That's why https://github.com/tigerbeetledb/tigerbeetle/releases has not contained any artifacts recently. They only contained artifacts when I created manual releases.

So I think we can solve this by putting the release.yml stuff into scheduled_release.yml and then saying `needs: scheduled_release` in the build/push artifact job. Let me know if you want to call to explain what I mean.

Reply:
Good catch! We can chat about this a bit later, but another possible solution is to make it so this release workflow is callable, and then `scheduled_release` calls it (eg, how `pipeline.yml` does things)

Reply:
Cool, sounds good!